### PR TITLE
PG Governor: bump up default number of compute

### PR DESCRIPTION
**Pull request by nzhang1220**

### CODE REVIEW

**Comment by nzhang1220** [comment]
@schwankl open a new PR, which refers to our discussion in https://github.com/oneops/circuit-oneops-1/pull/370. Not quite sure if this is the right direction, but would be happy to get your feedback.

File: packs/postgresql-governor.rb [hunk 1]
```diff
@@ -265,11 +265,59 @@
...
+                ]
+            }
+            ]
+        }
+        ]
+    }'
 }
+}
+
+#resource "lb",
```
Comment (schwankl) [inline packs/postgresql-governor.rb]: rm old or add comment to why

**Comment by nzhang1220** [comment]
@schwankl remove the old `lb` and hope this will look a little better.

**Comment by nzhang1220** [comment]
@schwankl would it be nice if you may have a couple of minute to review the change? Thanks 

(I am just curious about learning how to correctly override a relation in the application pack level)

**Pull request merged**